tests/test_pgexecute.py

```diff
@@ -69,16 +69,43 @@ def test_database_list(executor):
     databases = executor.databases()
     assert '_test_db' in databases
 
-@pytest.fixture(params=[True, False])
-def expanded(request):
-    return request.param
+@dbtest
+def test_invalid_syntax(executor):
+    with pytest.raises(psycopg2.ProgrammingError) as excinfo:
+        run(executor, 'invalid syntax!')
+    assert 'syntax error at or near "invalid"' in str(excinfo.value)
 
 @dbtest
-def test_unicode_support_in_output(executor, expanded):
-    if expanded:
+def test_invalid_column_name(executor):
+    with pytest.raises(psycopg2.ProgrammingError) as excinfo:
+        run(executor, 'select invalid command')
+    assert 'column "invalid" does not exist' in str(excinfo.value)
+
+@pytest.yield_fixture(params=[True, False])
+def expanded(request, executor):
+    if request.param:
```

Comment:
Took me a while to recognize this is a setup and teardown technique to toggle the expanded output. I'll probably add a comment after merge. :)